.env.sample

```
@@ -0,0 +1,11 @@
+PORT=portnumber
+
+DATABASE_URL="mysql://username:password@127.0.0.1:3306/database_name"
```

ip주소, portnumber도 다른 변수로 지정해주세요

예시입니다.
```suggestion
DATABASE_URL="mysql://username:password@hostnumber:portnumber/database_name"
```